src/overlays/actors/ovl_En_Holl/z_en_holl.c

```diff
@@ -2,26 +2,75 @@
 
 #define FLAGS ACTOR_FLAG_4
 
-// Horizontal Plane parameters
-
-#define PLANE_Y_MIN -50.0f
-#define PLANE_Y_MAX 200.0f
-
-#define PLANE_HALFWIDTH 100.0f
-#define PLANE_HALFWIDTH_2 200.0f
+// Horizontal holls parameters (`ENHOLL_H_*`)
+// All horizontal holls are cuboids which react to how far (depth) and in which direction (side) the player is from the
+// actor, in the actor's local z direction.
+
+// Defines the height of horizontal holls (all kinds) with a Y range relative to the actor position.
+#define ENHOLL_H_Y_MIN -50.0f
+#define ENHOLL_H_Y_MAX 200.0f
+
+// Defines the width of horizontal holls (all kinds),
+// with a half-width extending on both lateral sides (towards local +x and -x).
+#define ENHOLL_H_HALFWIDTH_NARROW 100.0f
+#define ENHOLL_H_HALFWIDTH 200.0f
```

Comment:
wonder if itd make sense to put these defines near where the action functions for the different types are (if it even works out that way order wise in the file)

the massive wall of 60 lines of defines at the top is kinda unfortunate, but not the end of the world lol

Reply:
We can also have no define at all <img src="https://cdn.discordapp.com/emojis/814280618959568896.webp?size=240&quality=lossless" alt="Sadge" height="24">
and just use the floats directly everywhere

Of course that's not what I'd prefer, I think it's easier to read (let alone modify) this actor with the defines

Some (most?) defines are used by more than one type, those can't really be properly placed near a function, so should stay at the top?
Then it would be weird to have defines a bit scattered around, at least to me it would

And tbh yeah it's not the end of the world, on my screen it's a screen full which is tolerable?